docs/source/release/v3.9.0/ui.rst

```diff
@@ -13,6 +13,7 @@ Instrument View
 ###############
 
 - New peak comparison tool on the pick tab. The user can select two peaks and information relating to their properties and the angles between them.
+- New tool for computing in-plane and out-of-plane angles in Qlab space for crystal alignment.
```

Comment:
No and nothing specific. I was going to add an entry [here](http://www.mantidproject.org/MantidPlot:_Instrument_View) after the PR went through.

theta -> elevation angle from plane defined by the three peaks to the 4th peak
phi -> in plane angle with reference to the beam direction

Actually I think you're correct and you could just use 2 + sample here.